src/deepsparse/benchmark/benchmark_model.py

```diff
@@ -81,6 +81,13 @@
    zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none \
    --input_shapes "[1,512],[1,512],[1,512]"
 
+##########
+Example on a OPT (Large Language Model) from SparseZoo with sequence length 256:
```

Comment:
Note: running the sparsezoo model is currently failing:

```bash
Traceback (most recent call last):
  File "/home/ubuntu/damian/deepsparse_venv/bin/deepsparse.benchmark", line 33, in <module>
    sys.exit(load_entry_point('deepsparse-nightly', 'console_scripts', 'deepsparse.benchmark')())
  File "/home/ubuntu/damian/deepsparse/src/deepsparse/benchmark/benchmark_model.py", line 481, in main
    result = benchmark_model(
  File "/home/ubuntu/damian/deepsparse/src/deepsparse/benchmark/benchmark_model.py", line 400, in benchmark_model
    model = compile_model(
  File "/home/ubuntu/damian/deepsparse/src/deepsparse/engine.py", line 942, in compile_model
    return Engine(
  File "/home/ubuntu/damian/deepsparse/src/deepsparse/engine.py", line 321, in __init__
    self._eng_net = LIB.deepsparse_engine(
RuntimeError: NM: error: Exception during initialization: /home/ubuntu/build/nyann/external/onnx-runtime/onnxruntime/core/optimizer/initializer.cc:43 onnxruntime::Initializer::Initializer(const onnx::TensorProto&, const onnxruntime::Path&) [ONNXRuntimeError] : 1 : FAIL : GetFileLength for /home/ubuntu/.cache/sparsezoo/neuralmagic/opt-1.3b-opt_pretrain-pruned50_quantW8A8/model.data failed:Invalid fd was supplied: -1
```

something wrong with referencing the external data by the overwritten model.

Reply:
@dbogunowicz were you able to look into this more?

Reply:
@dbogunowicz I think this might be my issue right now as well